pynamodb/attributes.py

```diff
@@ -257,3 +258,181 @@ def deserialize(self, value):
         Takes a UTC datetime string and returns a datetime object
         """
         return parse(value, dayfirst=False).datetime
+
+
+class MapAttribute(Attribute):
```

Comment:
so when I want to define an MapAttribute that requires verification, I need to create a subclass, otherwise I can use MapAttribute directly, right?